selenium-system-test/src/test/scala/htsstride/pages/StrideSignInPage.scala

```diff
@@ -47,4 +54,15 @@ object StrideSignInPage extends Page {
     setFieldByName("signature", "valid")
     setFieldByName("roles", "hts helpdesk advisor")
   }
+
+  private def fillInSecureStrideDetails()(implicit driver: WebDriver): Unit = {
```

Comment:
There some common behaviour with the above method which you could factor out. You could do something like this:
```scala
def fillInStandardStrideDetails()(implicit driver: WebDriver): Unit =
   fillInStrideDetails("hts helpdesk advisor")

def fillInSecureStrideDetails()(implicit driver: WebDriver): Unit =
   fillInStrideDetails("hts helpdesk advisor secure")

private def fillInStrideDetails(role: String)(implicit driver: WebDriver): Unit = {
     setFieldByName("pid", "pid1234")
     setFieldByName("usersGivenName", "test-given-name")
     setFieldByName("usersSurname", "test-surname")
     setFieldByName("emailAddress", "test@hmrc-hts.com")
     setFieldByName("status", "true")
     setFieldByName("signature", "valid")
     setFieldByName("roles", role)
   }
```